prediction_market_agent/agents/safe_guard_agent/evals/data/big_safes.csv

```diff
@@ -0,0 +1,715 @@
+safe_address,total_transferred_usd
```

Comment:
Please add some info to a README here (`/evals/`) describing what the data contains and a tiny excerpt of the data. Helps external contributors :)